src/XTModule.h

```diff
@@ -91,7 +91,9 @@ struct XTModule : public rack::Module
     {
         std::string dataPath = SurgeStorage::skipPatchLoadDataPathSentinel;
 
+#ifndef USING_CARDINAL_NOT_RACK
```

Comment:
This change looks super wrong. It will force a wavetable load and scan in every module when only two need them. What are you trying to accomplish with this?

Reply:
So gotcha - this is not the way to do it then

Surge storage suppresses a bunch of optional loads if the data path is the sentinel (which is why this is a string not an fs path) and so basically you’ve added loads of disk io and memory to fix a log print :) 

I think the thing to do is suppress the print if wavetables aren’t loaded not to force every module to load all the wavetables!

Reply:
oops! ok I will have the commit without this specific change then, the weird path just threw me off and I immediately thought it was an issue.

Reply:
Yeah if you want to move the if to the print that’s fine too. Or heck even remove those prints - I put them in to debug 4 years ago or whatever and have not used them since

Reply:
it is fine, they can be handy in some cases. I force-pushed to the PR without this bad change on it, now it is only theme related